core/chains/evm/gas/rollups/custom_calldata_da_oracle.go

```diff
@@ -0,0 +1,164 @@
+package rollups
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"math/big"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/assets"
+	evmclient "github.com/smartcontractkit/chainlink/v2/core/chains/evm/client"
+	evmconfig "github.com/smartcontractkit/chainlink/v2/core/chains/evm/config"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/config/oracletype"
+)
+
+type customCalldataDAOracle struct {
+	services.StateMachine
+	client     l1OracleClient
```

Comment:
nit: call this daOracleClient

Reply:
@matYang this corresponds to the `l1OracleClient` interface so updating this will change several of the other oracle files and trigger some mockery changes. In addition to `l1_oracle_client.go` there's also `l1_oracle.go`. 

I think maybe it's better to rename them both in a separate PR, what do you think? I created this ticket: https://smartcontract-it.atlassian.net/browse/CCIP-3743